Sign-up.dart

```diff
@@ -0,0 +1,119 @@
+import 'dart:io';
+import 'dart:convert';
+import 'Welcome.dart';
+import 'home.dart';
+// ignore: unused_import
+import 'package:crypt/crypt.dart';
+
+class Sing_up {
+  Sing() async {
+    int count = 0;
+    print("Welcom at Sing Up page");
+    print("--------------------------");
+    print("Please put your information");
+    //..............................................
+    stdout.write("a. FullName:");
+    String? name = stdin.readLineSync();
+    if (name!.isEmpty) {
+      print("This field is required");
+      count++;
+    }
+    if (count > 1) {
+      home g = new home();

```

Comment:
as I mentioned earlier, define another entity (Coordinator) that handles the transitions between the screens.